docker/Dockerfile.ng

```diff
@@ -0,0 +1,213 @@
+FROM usualoma/ruby-with-therubyracer:2.4.0-alpine
```

Comment:
@usualoma is an absolute legend. Pushed 2.2.x/2.3.x images already!

Will check them out this evening/when I get a spare second to work on this again.